### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by atoshmukhamedov**

### CODE REVIEW

File: cron/cron.ts [hunk 1]
```diff
@@ -93,51 +94,25 @@ export async function daily(bot: Bot<BotContext>) {
...
+      //   await bot.api
+      //     .sendPhoto(user.userId, file, {
+      //       caption: message + (randomHadith ? `\n\n${randomHadith.content}` : ''),
+      //       reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
+      //     })
+      // } else {
+      try {
+        await bot.api.sendMessage(user.userId, message + (randomHadith ? `\n\n${randomHadith.content}` : ''), {
+          reply_markup: { keyboard: buttons.build(), resize_keyboard: true },
+        })
```
Comment (Xayrulloh) [inline cron/cron.ts:107]: wuni tegidan statusini true qib qo'yiw keremasmi

File: cron/cron.ts [hunk 2]
```diff
@@ -93,51 +94,25 @@ export async function daily(bot: Bot<BotContext>) {
...
+      } catch (error) {
+        await handleSendMessageError(error, user)
```
Comment (Xayrulloh) [inline cron/cron.ts:109]: what it does

File: helper/errorHandler.ts
Comment (Xayrulloh) [inline helper/errorHandler.ts:1]: good work

File: utils/env.ts [hunk 3]
```diff
@@ -20,7 +20,6 @@ export const schema = z.object({
   WEBHOOK_URL: z.string().url().optional(),
   WEBHOOK_ENABLED: z.enum(['true', 'false']).transform((v) => JSON.parse(v)),
   WEBHOOK_PORT: z.number({ coerce: true }),
-  LIMIT: z.number({ coerce: true }),
```
Comment (Xayrulloh) [inline utils/env.ts:23]: why

**Review by Xayrulloh — CHANGES_REQUESTED** [review]

File: cron/cron.ts (hunk 1 above)
Comment (atoshmukhamedov) [reply cron/cron.ts:107]: could you explain why it was set to false

File: cron/cron.ts (hunk 2 above)
Comment (atoshmukhamedov) [reply cron/cron.ts:109]: it handles error)

Comment (atoshmukhamedov) [reply cron/cron.ts:109]: Its purpose is reduce duplicate code

File: utils/env.ts (hunk 3 above)
Comment (atoshmukhamedov) [reply utils/env.ts:23]: unused

File: cron/cron.ts (hunk 1 above)
Comment (Xayrulloh) [reply cron/cron.ts:107]: agar block qigan bo'sa status = false, agar succes bo'sa status = true ya'ni bilbo'midiku uni status oldin false bo'gan boliwi mumkinde

File: cron/cron.ts (hunk 2 above)
Comment (Xayrulloh) [reply cron/cron.ts:109]: boya ko'rib cundim, faqat u functionga yetib bormaganidm wunicun yozudm

File: utils/env.ts (hunk 3 above)
Comment (Xayrulloh) [reply utils/env.ts:23]: from now?

File: cron/cron.ts (hunk 1 above)
Comment (atoshmukhamedov) [reply cron/cron.ts:107]: is it right time to meeting

File: helper/errorHandler.ts
Comment (atoshmukhamedov) [reply helper/errorHandler.ts:1]: Can I resolve conversation?

**Review by Xayrulloh — APPROVED** [review]

**Pull request merged**